src/bindings/c/ov/tests/CMakeLists.txt

```diff
@@ -6,13 +6,13 @@ set(TARGET_NAME "OpenVinoCAPITests")
 
 # Find OpenCV components if exist
 find_package(OpenCV COMPONENTS core imgproc imgcodecs QUIET)
-if(NOT OpenCV_FOUND)
-    message(WARNING "OPENCV is disabled or not found, ${TARGET_NAME} is disabled")
-    return()
-endif()
 
 add_executable(${TARGET_NAME} ov_c_api_test.cpp test_model_repo.hpp)
 
+if(OpenCV_FOUND)
```

Comment:
Let's remove opencv dependency from c tests

Reply:
yes, PR12187 does it.